qutebrowser/browser/commands.py

```diff
@@ -853,7 +853,7 @@ def zoom_out(self, count=1):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('count', count=True)
-    def zoom(self, zoom: int=None, count=None):
+    def zoom(self, zoom: int = None, count=None):
```

Comment:
Not sure what happened here - it's already like that in `master`, and this change also doesn't show up in the "Files changed" tab.